theia-extensions/viewer-prototype/src/browser/trace-server-preference.ts

```diff
@@ -1,8 +1,11 @@
 import { PreferenceSchema, PreferenceProxy, PreferenceScope } from '@theia/core/lib/browser';
-import { TRACE_SERVER_DEFAULT_PORT } from '../common/trace-server-url-provider';
+import { TRACE_VIEWER_DEFAULT_PORT } from '../common/trace-server-url-provider';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { traceServerDefaultArgs } = require('../../package.json');
```

Comment:
I'm not for providing it in the package.json of the theia-trace-extension. It practically the same as having it as constant in the source code. 

I suggested to have the default arguments an empty string and it can be part of the source code.